bam-cmd/src/main/java/org/victorchang/QnameCommand.java

```diff
@@ -120,26 +120,33 @@ private boolean createIndexDirectory(Path bamPath, boolean force) {
         if (indexDirectory == null) {
             indexDirectory = QnameCommand.getDefaultIndexPath(bamPath);
         }
-        return createDirectory(indexDirectory, force);
+        if (createDirectory(indexDirectory)) {
+            Path indexLevel1 = indexDirectory.resolve(IndexVersion.VERSION0.fileName("1"));
```

Comment:
We should probably create a method `IndexVersion.latestVersion()` and use it everywhere - so that when the new version is added, we only need to update enum class